tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py

```diff
@@ -419,33 +419,6 @@ def test_stable_diffusion_pipeline_with_model_offloading(self):
         for module in pipe.text_encoder, pipe.unet, pipe.vae:
             assert module.device == torch.device("cpu")
 
-    def test_img2img_2nd_order(self):
```

Comment:
this is actually also quite an important test. Could we try to just run it for 5 inference steps maybe instead of 10?